app/javascript/javascript/params.js

```diff
@@ -5,5 +5,41 @@ export const getParams = () =>
     .filter(x => x.length !== 0)
     .map(x => x.split("="))
     .map(([k, v]) => [k, decodeURIComponent(v)])
-    .map(([k, v]) => [k, JSON.parse(v)])
+    .map(([k, v]) => [k, parseIfJSON(v)])
     .reduce((acc, [k, v]) => ({ ...acc, [k]: v }), {});
+
+const parseIfJSON = value => {
+  if (isJSON(value)) {
+    return JSON.parse(value);
+  } else {
+    return value;
+  }
+};
+
+export const isJSON = text => {
+  if (
```

Comment:
I just copied this regex as a recommended way to check if something is a json :)